integration-tests/base64/src/test/java/org/apache/camel/quarkus/component/base64/it/Base64Test.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.camel.quarkus.component.base64.it;
+
+import io.quarkus.test.junit.QuarkusTest;
+import io.restassured.RestAssured;
+import io.restassured.http.ContentType;
+import io.restassured.response.ExtractableResponse;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+@QuarkusTest
+class Base64Test {
+
+    @Test
+    public void test() {
+        byte[] body;
+
+        ExtractableResponse response = RestAssured.given() //
+                .contentType(ContentType.BINARY).body("Hello World".getBytes()).post("/base64/post") //
```

Comment:
String.getBytes() relies on the default encoding of the given platform and therefore this test will fail on exotic platforms that have some ASCII incompatible default encoding set. 
```suggestion
                .contentType(ContentType.BINARY).body("Hello World".getBytes("UTF-8")).post("/base64/post") //
```